irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -32,59 +35,17 @@
 #include "utils/string_builder.hpp"
 
 namespace {
-  struct PreparedStatement {
-    std::string command_name;
-    std::string command_base;
-    std::vector<std::string> permission_checks;
-
-    static const std::string validationPrefix;
-    static const std::string noValidationPrefix;
-  };
-
-  const std::string PreparedStatement::validationPrefix = "WithValidation";
-  const std::string PreparedStatement::noValidationPrefix = "WithOutValidation";
-
-  // Transforms prepared statement into two strings:
-  //    1. SQL query with validation
-  //    2. SQL query without validation
-  std::pair<std::string, std::string> compileStatement(
-      const PreparedStatement &statement) {
-    // Create query with validation
-    auto with_validation = boost::format(statement.command_base)
-        % (statement.command_name + PreparedStatement::validationPrefix);
-
-    // append all necessary checks to the query
-    for (const auto &check : statement.permission_checks) {
-      with_validation = with_validation % check;
-    }
-
-    // Create query without validation
-    auto without_validation = boost::format(statement.command_base)
-        % (statement.command_name + PreparedStatement::noValidationPrefix);
-
-    // since checks are not needed, append empty strings to their place
-    for (size_t i = 0; i < statement.permission_checks.size(); i++) {
-      without_validation = without_validation % "";
-    }
-
-    return {with_validation.str(), without_validation.str()};
-  }
-
-  void prepareStatement(soci::session &sql,
-                        const PreparedStatement &statement) {
-    auto queries = compileStatement(statement);
+  static constexpr size_t kRolePermissionSetSize =
```

Comment:
I guess we could declare these variables without `static` because they're already in anonymous namespace.